storage/object/api-cli/lifecycle-rules-api.mdx

```diff
@@ -189,9 +189,9 @@ The lifecycle configuration above translates to the following effective rules:
 
 ## Setting rules for incomplete multipart uploads
 
-When using the `awscli` to perform a [multipart upload](/storage/object/api-cli/multipart-uploads/) it is possible to set up a bucket lifecycle rule that allows you to automate the *stoppage* of uploads if they are not completed. You can define a deadline for the expected completion with a lifecycle rule. If the multipart upload is incomplete by the end of the defined timeframe, it will be stopped and the parts associated with it are deleted. 
+When using the `awscli` to perform a [multipart upload](/storage/object/api-cli/multipart-uploads/) it is possible to set a bucket lifecycle rule that allows you to automate the *stoppage* of uploads if they are not completed. You can define a deadline for the expected completion with a lifecycle rule. If the multipart upload is incomplete by the end of the defined timeframe, it will be stopped and the parts associated with it will be deleted. 
```

Comment:
```suggestion
When using the `awscli` to perform a [multipart upload](/storage/object/api-cli/multipart-uploads/), it is possible to set a bucket lifecycle rule that allows you to automate the *stoppage* of uploads if they are not completed. You can define a deadline for the expected completion with a lifecycle rule. If the multipart upload is incomplete by the end of the defined timeframe, it will be stopped and the parts associated with it will be deleted. 
```